### PR TITLE
fix(core,platform): prepare tokenizer and approval flow for translation

### LINKED ISSUE #8731: <fd-tokenizer> 'moreTerm' string is not localized
#### Is this a bug, enhancement, or feature request?
A bug
#### Briefly describe your proposal.
Need to localize this string

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
We're using Angular 13 and F-NGX 0.35.4

#### If this is a bug, please provide steps for reproducing it.
Using the `<fd-tokenizer>` component, add in more tokens so that they exceed the input field width, then focus out from the field until you see the `xxx more` string displayed inside the input field of the tokenizer

Screen Shot 2022-09-22 at 10 56 11 AM

#### Please provide relevant source code if applicable.
https://github.com/SAP/fundamental-ngx/blob/main/libs/core/src/lib/token/tokenizer.component.ts#L111

#### Is there anything else we should know?
This will cause translation issues with the `<fdp-multi-combobox>` component since itself is using the `<fd-tokenizer>`

### PR DESCRIPTION
**Pull request by mikerodonnell89**
fixes #8731, #5837, #6238, #8492

Prepares the tokenizer and approval flow components for translation

### CODE REVIEW

File: libs/i18n/src/lib/languages/georgian.ts [hunk 1]
```diff
@@ -212,7 +215,11 @@ export const FD_LANGUAGE_GEORGIAN: FdLanguage = {
         toolbarEditApprover: 'შეასწორე დამდასტურებელი',
         watchersInputPlaceholder: 'ძებნა...',
         userListSelectedItemsCountSingular: '1 ელემენტი არჩეულია',
-        userListSelectedItemsCountPlural: '{{ count }} ელემენტი არჩეულია'
+        userListSelectedItemsCountPlural: '{{ count }} ელემენტი არჩეულია',
+        statusApproved: 'approved',
+        statusRejected: 'rejected',
+        statusInProgress: 'in progress',
+        statusNotStarted: 'not started'
```
Comment (g-cheishvili) [inline libs/i18n/src/lib/languages/georgian.ts:222]:
```suggestion
        userListSelectedItemsCountPlural: '{{ count }} ელემენტი არჩეულია',
        statusApproved: 'დადასტურებული',
        statusRejected: 'უარყოფილი',
        statusInProgress: 'მიმდინარე',
        statusNotStarted: 'არ დაწყებულა'
```

File: libs/i18n/src/lib/languages/georgian.ts [hunk 2]
```diff
@@ -131,6 +131,9 @@ export const FD_LANGUAGE_GEORGIAN: FdLanguage = {
         deleteButtonLabel: 'წაშლადი',
         ariaRoleDescription: 'ტოკენი'
     },
+    coreTokenizer: {
+        moreLabel: '{{count}} more'
```
Comment (g-cheishvili) [inline libs/i18n/src/lib/languages/georgian.ts:135]:
```suggestion
        moreLabel: '{{count}} სხვა'
```

File: libs/platform/src/lib/approval-flow/approval-flow-node/approval-flow-node.component.html [hunk 3]
```diff
@@ -63,7 +63,18 @@
...
+                }}</ng-container>
+                <ng-container *ngIf="node.status === 'rejected'">{{
+                    'platformApprovalFlow.statusRejected' | fdTranslate
+                }}</ng-container>
+                <ng-container *ngIf="node.status === 'in progress'">{{
+                    'platformApprovalFlow.statusInProgress' | fdTranslate
+                }}</ng-container>
+                <ng-container *ngIf="node.status === 'not started'">{{
+                    'platformApprovalFlow.statusNotStarted' | fdTranslate
+                }}</ng-container>
```
[the remark this answers is not in the document]
Comment (mikerodonnell89) [reply libs/platform/src/lib/approval-flow/approval-flow-node/approval-flow-node.component.html:77]: Can I ask why? We need a switch case either in the typescript, or here, I don't see one being preferable over the other

File: libs/i18n/src/lib/languages/albanian.ts [hunk 4]
```diff
@@ -137,6 +137,9 @@ export const FD_LANGUAGE_ALBANIAN: FdLanguage = {
         deleteButtonLabel: 'E fshishme',
         ariaRoleDescription: 'shenjë'
     },
+    coreTokenizer: {
+        moreLabel: '{{count}} more'
```
Comment (platon-rov) [inline libs/i18n/src/lib/languages/albanian.ts:141]: just an idea - maybe we can have some mark in not translated yet strings? like `[TODO]` so then when someone will be translating it it'll be easier to find these strings.

File: libs/platform/src/lib/approval-flow/approval-flow-node/approval-flow-node.component.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (mikerodonnell89) [reply libs/platform/src/lib/approval-flow/approval-flow-node/approval-flow-node.component.html:77]: We need a switch case either in the typescript, or here, I don't see one being preferable over the other

[the remark this answers is not in the document]
Comment (mikerodonnell89) [reply libs/platform/src/lib/approval-flow/approval-flow-node/approval-flow-node.component.html:77]: I do think it is cleaner and easier to read here in the template than the component TS, no need for translation resolver, no need for dealing with subscriptions in the typescript etc.

**Review by platon-rov — APPROVED** [review]

**Pull request merged**